### PR TITLE
add naming rules to contributing

### PR DESCRIPTION
**Pull request by sh-rp**
### Description

We want to add some rules how to name stuff (mostly branches). Let's discuss :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,6 +52,30 @@ We use **master** branch for hot fixes (including documentation) that needs to b
...
+```
+
+#### Branch categories
+
+* **feat** - a new feature that is being implemented (ticket required)
+* **fix** - a change that fixes a bug (ticket required)
+* **exp** - an experiment where we are testing a new  idea or want to demonstrate something to the team, might turn into a feat later (ticket encouraged)
+* **test** - anything related to the tests (ticket encouraged)
+* **blogs** - a new entry to our blog (ticket optional)
+* **docs** - a change to our docs (ticket optional)
```
Comment (sultaniman) [inline CONTRIBUTING.md:72]: also maybe we can add `infra` for things related to CI

Comment (sultaniman) [reply CONTRIBUTING.md:72]: should we also add `refactor` scope?

**Review by burnash — APPROVED** [review]

**Review by sultaniman — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -52,6 +52,30 @@ We use **master** branch for hot fixes (including documentation) that needs to b
...
+
+#### Ticket Numbers
+
+We encourage you to attach your branches to a ticket, if none exists, create one an explain what you are doing. For feat and fix branches, tickets are mandatory, for exp and test branches encouraged and for blogs and docs branches optional.
```
[the remark this answers is not in the document]
Comment (sultaniman) [reply CONTRIBUTING.md:76]: and maybe enclose feat/fix in backticks

**Pull request merged**